apps/bixarena/app/bixarena_app/model/error_handler.py

```diff
@@ -85,84 +85,84 @@ def handle_api_error_message(error: Exception) -> str:
     # 401 - Authentication Error
     if isinstance(error, AuthenticationError):
         return (
-            "**Connection Issue**\n\n"
-            "The service is currently unavailable.\n"
-            "Please start a new battle to try again.\n\n"
-            "_Error Code: 401_"
+            "<strong>Connection Issue</strong><br><br>"
+            "The service is currently unavailable.<br>"
+            "Please start a new battle to try again.<br><br>"
+            "<em>Error Code: 401</em>"
```

Comment:
A potential future improvement (out of scope of this PR): we could show the battle ID so users can share it when reporting issues.